### DISCUSSION TITLE
Filtering videos in the feed

### DISCUSSION THREAD

**Discussion by ghost** [FreeTubeApp/FreeTube#2798] [General]
First, I would like to thank you for creating a great app!

I was wondering, if it would be possible someday to add the ability to filter videos in the feed by length (I'm talking about videos from channels we are subscribed to).

As you know, for some time now many channels have started to upload short videos (less than 1 minute), which clogs the feed. It would be nice to have a filter.

Thank you for all the great work,
Andrea

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi there, thank you for loving the app. 

Several people have requested the same feature as u and it will eventually be implemented. It is just a matter of when and not if anymore.

Originally it was requested in https://github.com/FreeTubeApp/FreeTube/issues/2008 but is now closed because this feature is a subset of #1070
